mobile/status.go

```diff
@@ -1129,16 +1129,6 @@ func makeJSONResponse(err error) string {
 	return string(outBytes)
 }
 
-func AddPeer(enode string) string {
```

Comment:
We sure this endpoint is not used in desktop/mobile?

Reply:
In status-mobile it's defined in the cpp layer, but we don't use anymore, looks like a leftover from past refactors in mobile. https://github.com/status-im/status-mobile/blob/ece95ca674c31d696597eccbf700ece8244f12f5/modules/react-native-status/nodejs/status.cpp#L1892